lib/cdo/languages.rb

```diff
@@ -26,4 +26,24 @@ def self.get_hoc_locale_by_unique_language(unique_language)
   def self.get_hoc_unique_language_by_locale(locale)
     @@table.select(:unique_language_s, :locale_s).where("locale_s = '#{locale}'").first[:unique_language_s]
   end
+
+  def self.get_starwars_languages
+    @@table.select(:starwars_b, :crowdin_name_s)
+  end
+
+  def self.get_csf_languages
+    @@table.select(:csf_b, :crowdin_name_s)
+  end
+
+  def self.get_frozen_languages
+    @@table.select(:frozen_b, :crowdin_name_s)
+  end
+
+  def self.get_minecraft_adventurer_languages
+    @@table.select(:minecraft_adventurer_b, :crowdin_name_s)
+  end
+
+  def self.get_minecraft_designer_languages
```

Comment:
Nit: re-order these methods to be in the same order as in your supported.md file.